### PR TITLE
fix: Dutch additions

### PR DESCRIPTION
**Pull request by aleene**
More dutch additions to the taxonomies

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -1243,7 +1243,8 @@ agribalyse_proxy_food_name:en:Beer, dark
 agribalyse_proxy_food_name:fr:Bière brune
 
 <en:Beers
-en:India Pale Ale, IPA beers
+en:India Pale Ale (IPA), IPA beers
```
Comment (teolemon) [inline taxonomies/categories.txt:1246]:
```suggestion
en:India Pale Ale (IPA), IPA beers
fr:Bière IPA
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -62603,16 +62612,19 @@ fr:Sauces au miel
 <fr:Sauces au miel
 en:Honey dill sauces
 fr:Sauce à l'aneth au miel
+nl:Dillesauzen
 wikidata:en:Q17098070
 
 <en:Sauces
```
Comment (teolemon) [inline taxonomies/categories.txt:62618]:
```suggestion
<en:Sauces
en:Wine sauces
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -62603,16 +62612,19 @@ fr:Sauces au miel
...
 fr:Sauces au vin
+nl:Wijnsauzen
 agribalyse_proxy_food_code:en:11164
 agribalyse_proxy_food_name:en:Red wine sauce
 agribalyse_proxy_food_name:fr:Sauce au vin rouge
 
 <fr:Sauces au vin
```
Comment (teolemon) [inline taxonomies/categories.txt:62625]:
```suggestion
<en:Wine sauces
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -62603,16 +62612,19 @@ fr:Sauces au miel
...
 fr:Sauces au vin blanc, Sauce au vin blanc
+nl:Witte wijnsauzen
 
 <fr:Sauces au vin blanc
```
Comment (teolemon) [inline taxonomies/categories.txt:62629]:
```suggestion
<en:White wine sauces
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -62621,6 +62633,7 @@ wikidata:en:Q3474123
 <fr:Sauces au vin
```
Comment (teolemon) [inline taxonomies/categories.txt:62633]:
```suggestion

<en:Wine sauces
```

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -63098,7 +63114,7 @@ es:Aliolis, Salsas alioli, Alioli, Salsa alioli
 fi:Aiolit, Aiolikastikkeet
 fr:Aïolis, Aiolis, sauces Aïoli, sauces Aïolis, sauce Aïoli
 it:Salsa aioli
-nl:Aiolisauzen, Knoflooksauzen
+nl:KnoflooksauzenAiolisauzen
```
Comment (teolemon) [inline taxonomies/categories.txt:63117]:
```suggestion
nl:Knoflooksauzen, Aiolisauzen
```

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -92982,6 +92983,40 @@ protected_name_type:en: pdo
...
+en:Vegetarian balls
+fr:Boulettes végétariennes
+nl:Vegetarische balletjes
+
+<en:Vegetarian balls
+en:Falafels, Falafel
+de:Falafel
+es:Falafel
+fi:falafelit
+fr:Falafels, falafel, Boulettes aux pois-chiche et aux fèves frites
```
Comment (teolemon) [inline taxonomies/categories.txt:92996]:
```suggestion
fr:Falafels, falafel, Boulettes aux pois-chiche frites,  Boulettes aux fèves frites, Boulettes de pois-chiche frits,  Boulettes de fèves frites

```

File: taxonomies/nutrients.txt [hunk 8]
```diff
@@ -1008,6 +1008,11 @@ unit:en: g
 wikidata:en: Q420731
 wikipedia:en: https://en.wikipedia.org/wiki/Melissic_acid
 
+zz:unsaturated-fat
```
Comment (teolemon) [inline taxonomies/nutrients.txt:1011]: what is zz: as opposed to xx: ?

Comment (aleene) [reply taxonomies/nutrients.txt:1011]: no idea

**Comment by aleene** [comment]
@stephanegigandet can you have a look at these test fails. It seems that the order of the result strings changed. Did I miss an update (merge) somewhere?

**Comment by stephanegigandet** [comment]
> @stephanegigandet can you have a look at these test fails. It seems that the order of the result strings changed. Did I miss an update (merge) somewhere?

The order changed because the number of levels under some of those categories changed. The field is ordered with categories with the most levels under them first.

**Comment by aleene** [comment]
Where can the test results be found?

**Comment by aleene** [comment]
Giving up, I do not know where or what I should change. I hate these tests

**Comment by aleene** [comment]
I can do the conflicts

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -70226,12 +70253,20 @@ nl:Diepvriesfalafel
...
 pt:Almôndegas
 
 <en:Meat balls
+<en:Chicken
 en:Chicken meat balls
 fr:Boulettes au poulet
+nl:Kippegehaktballen
+
+<en:Meat balls
+<en:Chicken
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:70266]:
```suggestion
```

Comment (stephanegigandet) [reply taxonomies/categories.txt:70266]: Thanks!

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**